sql/2020/16_Ecommerce/percent_of_mobile_ecommsites_using_each_cmp.sql

```diff
@@ -0,0 +1,32 @@
+#standardSQL
+# 13_17b: % Share of CMPs on mobile eCommerce origins
+SELECT
+  _TABLE_SUFFIX AS client,
+  app,
+  COUNT(DISTINCT url) AS freq,
+  total,
+  COUNT(DISTINCT url) / total AS pct
+FROM
+  `httparchive.technologies.2020_08_01_*`
+JOIN (
+  SELECT
+    _TABLE_SUFFIX,
+    COUNT(DISTINCT url) AS total
+  FROM
+    `httparchive.technologies.2020_08_01_*`
+  WHERE
+     category = 'Ecommerce'
+  GROUP BY
+    _TABLE_SUFFIX
+    )
+USING
+  (_TABLE_SUFFIX)
+WHERE
+  category = 'Cookie compliance'
+  and _TABLE_SUFFIX = 'mobile'
+GROUP BY
+  client,
+  app,
+  total
+order by
+  pct desc
```

Comment:
Nit: files should end in newlines for obscure reasons:

```suggestion
  pct desc

```